plugins/workload-management/src/main/java/org/opensearch/plugin/wlm/WorkloadManagementPlugin.java

```diff
@@ -112,6 +114,7 @@ public Collection<Object> createComponents(
         IndexNameExpressionResolver indexNameExpressionResolver,
         Supplier<RepositoriesService> repositoriesServiceSupplier
     ) {
+        workloadManagementSettings = new WorkloadManagementSettings(environment.settings(), clusterService.getClusterSettings());
```

Comment:
This will only work when the **server** codebase settings are initialised first. How are we verifying that it will be able to pull the settings defined by core `WorkloadManagementSettings`.